crates/ironrdp-client/src/gui.rs

```diff
@@ -63,9 +63,17 @@ impl GuiContext {
             match event {
                 Event::WindowEvent { window_id, event } if window_id == window.id() => match event {
                     WindowEvent::Resized(size) => {
+                        let scale_factor = (window.scale_factor() * 100.0) as u32;
+                        // TODO: it should be possible to get the physical size here, however winit doesn't make it straightforward.
+                        // FreeRDP does it based on DPI reading grabbed via [`SDL_GetDisplayDPI`](https://wiki.libsdl.org/SDL2/SDL_GetDisplayDPI):
+                        // https://github.com/FreeRDP/FreeRDP/blob/ba8cf8cf2158018fb7abbedb51ab245f369be813/client/SDL/sdl_monitor.cpp#L250-L262
+                        let (physical_width, physical_height) = (0, 0);
```

Comment:
question: Can’t you use this? https://docs.rs/winit/latest/winit/window/struct.Window.html#method.inner_size
Is it really fine to feed in 0s?

Reply:
`inner_size` gives you a [`PhysicalSize`](https://docs.rs/winit/latest/winit/dpi/struct.PhysicalSize.html), but that only gives you a "size represented in physical pixels." However without knowing the DPI (or other relation to a unit of length), we can't calculate the "physical size" that RDP wants, which is the width/height in millimeters. 

If we hand back `0`s here, they're simply ignored [per the spec](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpedisp/ea2de591-9203-42cd-9908-be7a55237d1c):

<img width="868" alt="image" src="https://github.com/Devolutions/IronRDP/assets/13578537/7800a1bb-6b5d-4d3d-85ce-ba621f341b34">

Reply:
@ibeckermayer 
I think passing in client `(0, 0)` is not optimal (it is not obvious that they will be ignored), maybe it would be great to change `RdpInputEvent::Resize` fields to have `physical_dimensions: Option<(u32, u32)>` instead of separate u32 fields? This will keep this implementation detail as close to PDU encode/decode code as possible

Reply:
This could be fixed in follow-up PR, merging PR to unblock @ibeckermayer